Makefile

```diff
@@ -31,6 +31,7 @@ DOCKER_BUILD = docker build --no-cache \
 	    -f slave.mk \
 	    PLATFORM=$(PLATFORM) \
 	    SKU=$(SKU) \
+            DEBUG_BUILD=$(DEBUG_BUILD) \
```

Comment:
don't quite get you. In default we are having this option off now. That should be the correct default behavior right?

---
In reply to: [99479772](https://github.com/Azure/sonic-buildimage/pull/247#discussion_r99479772) [](ancestors = 99479772)